Snowdog_Components/components/Organisms/footer/footer.hbs

```diff
@@ -47,9 +55,15 @@
                         </li>
                     {{/each}}
                 </ul>
+                
                 <div class="footer__switchers">
```

Comment:
we don't want `footer__switchers` to show if there's no language and currency so I'd suggest adding also this condition:
```suggestion
                {{#if (or language currency)}}
                    <div class="footer__switchers">
```